packages/patternlab/styleguide/source/_patterns/01-atoms/09-media/picture-element.json

```diff
@@ -1,18 +1,18 @@
 {
   "picture": {
-    "defaultImage": "../../assets/images/placeholder/400x300.png",
+    "defaultImage": "../../assets/images/placeholder/1600x600.png",
     "srcSets": [
       {
-        "url": "../../assets/images/placeholder/900x700.png",
-        "width": "1050"
+        "url": "../../assets/images/placeholder/500x300.png",
+        "width": "480"
       },
       {
-        "url": "../../assets/images/placeholder/800x450.png",
-        "width": "780"
+        "url": "../../assets/images/placeholder/900x700.png",
+        "width": "910"
       },
       {
-        "url": "../../assets/images/placeholder/600x450.png",
-        "width": "620"
+        "url": "../../assets/images/placeholder/1600x600.png",
+        "width": "1200"
       }
     ]
```

Comment:
the order of these images and breakpoints should be reversed

Reply:
done